lectures/lln_clt.md

```diff
@@ -78,7 +78,7 @@ print(X)
 ```
 
 In this setting, the LLN tells us if we flip the coin many times, the fraction
-of heads that we see will be close to the mean $p$.
+of heads that we see will be close to the mean $p$. We use $n$ to represent the number of times the coin is flipped.
```

Comment:
@SylviaZhaooo Would you mind to put the second sentence on a separate line -- this is our (unusual) convention for the lectures.

Reply:
Sure!